chromium_src/components/content_settings/core/common/cookie_settings_base.cc

```diff
@@ -171,6 +173,11 @@ bool CookieSettingsBase::IsCookieAccessAllowed(
   return IsChromiumCookieAccessAllowed(url, site_for_cookies, top_frame_origin);
 }
 
+bool CookieSettingsBase::isThirdParty(const GURL& url,
```

Comment:
I'm not exactly sure what this does, because I cannot find the definition of `IsThirdPartyRequest`, but methods should start with a capital letter in Chromium.

Reply:
`IsThirdPartyRequest` is a static function in the original cookie_settings_base.cc. I want to reuse it `inContentSettingsManagerImpl`, so I put it inside `CookieSettingsBase::isThirdParty`.